src/game/g_main.c

```diff
@@ -4296,7 +4296,8 @@ void CheckExitRules(void)
 				}
 				else
 				{
-					if (g_suddenDeath.integer && DynamiteOnObjective() && g_gametype.integer != GT_WOLF_STOPWATCH)
+					// sudden death is disabled on second round of stopwatch
+					if (g_suddenDeath.integer && DynamiteOnObjective() && (g_gametype.integer != GT_WOLF_STOPWATCH || g_currentRound.integer))
```

Comment:
Note to self: actually I should move the `DynamiteOnObjective()` check after for optimization.